src/apiCalls/accounts/getAccountFromApi.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { apiAddressSelector } from 'reduxStore/selectors';
+import { store } from 'reduxStore/store';
+import { addressEndpoint } from 'apiCalls/endpoints';
+import { AccountType } from 'types';
+
+export async function getAccountFromApi(address?: string) {
+  if (!address) {
+    return null;
+  }
+  const apiAddress = apiAddressSelector(store.getState());
+  const cleanApiAddress = apiAddress.endsWith('/')
```

Comment:
it seems like this part is being copy-pasted in multiple files. I would create a new selector here, something like cleanApiAddressSelector